src/tree.rs

```diff
@@ -33,16 +33,9 @@ pub struct Tree<Pane> {
 
     /// All the tiles in the tree.
     pub tiles: Tiles<Pane>,
-}
 
-impl<Pane> Default for Tree<Pane> {
-    // An empty tree
-    fn default() -> Self {
-        Self {
-            root: None,
-            tiles: Default::default(),
-        }
-    }
+    /// The previous id used to draw the tree.
```

Comment:
This doc comment should be changed to better show that it's a fixed value. When working on this I initially had this value be updated each draw call.